clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -401,6 +401,8 @@ static bool initTargetOptions(DiagnosticsEngine &Diags,
   Options.UniqueBasicBlockSectionNames =
       CodeGenOpts.UniqueBasicBlockSectionNames;
   Options.TLSSize = CodeGenOpts.TLSSize;
+  // TODO: Add correct codegen options in LLVM
+  // Options.TLSDesc = CodeGenOpts.getDefaultTLSDialect();
```

Comment:
This should be supported in this patch, otherwise there is no point in ignoring a rarely-used GCC option.

Reply:
Agreed.